docs/docs/services.md

```diff
@@ -645,14 +645,14 @@ So `validateUniqueness()` first tries to find a record with the given fields, an
 > **Why use this when the database can verify uniqueness with a UNIQUE INDEX database constraint?**
 >
 > You may be in a situation where you can't have a unique index (supporting a legacy schema, perhaps), but still want to make sure the data is unique before proceeding. There is also the belief that you shouldn't have to count on the database to validate your data—that's a core concern of your business logic, and your business logic should live in your Services in a Redwood app.
-> 
-> Another issue is that the error raised by Prisma when a record validates a unique index is swallowed by GraphQL and so you can't report it to the user (there are still ways around this, but it involves catching and re-throwing a different error). The error raised by `validateUniqueness()` is already safe-listed and allowed to be sent to the browser. 
+>
+> Another issue is that the error raised by Prisma when a record validates a unique index is swallowed by GraphQL and so you can't report it to the user (there are still ways around this, but it involves catching and re-throwing a different error). The error raised by `validateUniqueness()` is already safe-listed and allowed to be sent to the browser.
 
 #### Arguments
 
 1. The name of the db table accessor that will be checked (what you would call on `db` in a normal Prisma call). If you'd call `db.user` then this value is `"user"`.
 2. An object, containing the db fields/values to check for uniqueness, like `{ email: 'rob@redwoodjs.com' }`. Can also include additional options explained below that provide for a narrower scope for uniqueness requirements, and a way for the record to identify itself and not create a false positive for an existing record.
-3. [Optional] An object with options.
+3. [Optional] An object with options. `message` - custom error message. `db` - custom instance of the PrismaClient to use
```

Comment:
Is there another place to update docs and does this feel like enough?

Is it worth having something in here about how if you don't supply your own db in the options, a generic PrismaClient generated from your `prisma.schema` file will be used? The gotcha for me was that I was customizing my DATABASE_URL, and the validateUniqueness call wasn't using the right database.

Reply:
I think that's the only place for the docs. I would add a code snippet example below that, similar to how I did for showing a custom message.

Right before that code snippet would be a good place to explain why you'd want to pass your own in. You could maybe add it in an `:::info` or `:::caution` block so that it gets highlighted and people hopefully notice!

Reply:
Done